teamone/urls.py.orig

```diff
@@ -0,0 +1,62 @@
+"""teamone URL Configuration
```

Comment:
I don't think this file should even be in git? 
I may be wrong. 
Either way, there are some unresolved merge conflicts here

Reply:
done